test_project/README.md

```diff
@@ -0,0 +1,16 @@
+# test_project
```

Comment:
I think this whole `test_project` shouldn't be here.

Reply:
Done